modules/administration/pages/iss.adoc

```diff
@@ -2,29 +2,29 @@
 = Inter-Server Synchronization
 
 If you have more than one {productname} installation, you will probably want to ensure that they stay aligned on content and permissions.
-Inter-Server Synchronization (ISS) allows you to connect two or more {productname} Servers and keep them up-to-date.
+Inter-Server Synchronization (ISS) allows you to connect two or more {productname} servers and keep them up-to-date.
 
-To set up ISS, you need to define one {productname} Server as a master, with the other as a slave.
+To set up ISS, you need to define one {productname} server as a master, with the other as a slave.
 If conflicting configurations exist, the system will prioritize the master configuration.
 
 .Procedure: Setting up an ISS Master
 
 . In the {productname} {webui}, navigate to menu:Admin[ISS Configuration > Slave Setup], and click btn:[Add new master].
-. In the [guimenu]``Details for new Master`` dialog, provide these details for the Server to use as the ISS master:
+. In the [guimenu]``Details for new Master`` dialog, provide these details for the server to use as the ISS master:
 
 * In the [guimenu]``Master Fully-Qualified Domain Name`` field, enter the FQDN of the ISS master (for example: [systemitem]``http://server1.example.com``).
 * In the [guimenu]``Filename of this Master's CA Certificate`` field, enter the absolute  path to the CA certificate on the ISS master (for example: [systemitem]``/etc/pki/trust/anchors-org-ssl``).
-. Click btn:[Add new master] to add the ISS master.
+. To add the ISS master, click btn:[Add new master].
 
 .Procedure: Setting up an ISS Slave
 
 . In the {productname} {webui}, navigate to menu:Admin[ISS Configuration > Master Setup], and click btn:[Add new slave].
-. In the [guimenu]``Edit Slave Details`` dialog, provide these details for the Server to use as the ISS slave:
+. In the [guimenu]``Edit Slave Details`` dialog, for the server to use as the ISS slave provide these details:
 
 * In the [guimenu]``Slave Fully-Qualified Domain Name`` field, enter the FQDN of the ISS slave (for example: [systemitem]``http://server2.example.com``).
-* Check the [guimenu]``Allow Slave to Sync?`` checkbox to enable the slave to synchronize with the master.
-* Check the [guimenu]``Sync All Orgs to Slave?`` checkbox to synchronize all organizations to this slave.
-. Click btn:[Create] to add the ISS slave.
+* To enable the slave to synchronize with the master, check [guimenu]``Allow Slave to Sync?``.
+* To synchronize all organizations to this slave, check the [guimenu]``Sync All Orgs to Slave?`` checkbox.
+. To add the ISS slave, click btn:[Create].
```

Comment:
Why did you turn these around? Steps should start with a verb or a location ...

Reply:
> Why did you turn these around? Steps should start with a verb or a location ...
I forgot that rule, which kind of makes sense.  Reverting...  I fear there is more of this.

Reply:
hopefully reverted correctly.